packages/app/cypress/e2e/settings.cy.ts

```diff
@@ -13,22 +17,43 @@ describe('App: Settings', () => {
     cy.findByText('Settings').click()
 
     cy.get('div[data-cy="app-header-bar"]').should('contain', 'Settings')
-    cy.findByText('Device Settings').should('be.visible')
-    cy.findByText('Project Settings').should('be.visible')
+    cy.findByText(i18n.device.title).should('be.visible')
```

Comment:
I didn’t think that we made a clear decision to switch to the i18n shortcuts in our e2e tests. I know some of the e2e team disagreed with this strategy.

Reply:
I believe the slack thread was quite clear. Until I left it and then probably it was not. 
Do you want me to remove all this from the e2e? It is not part of this fix. 
I am just an amazing mistyper and this helps me use autocomplete.

Reply:
I agree we should not use the i18n in test. That said, it looks like many of the tests still do - rather than block this PR, I'd like to see it merged then a follow up PR to update all tests to not use i18n in the tests, if that's the path we are going down.

Reply:
Just was writing some tests, I really want to move away from the i18n labels asap, it is so hard to read!

```js
    cy.findByText(defaultMessages.migration.wizard.step1.button).click()
```

or..
 
```js
    cy.findByText(`Rename the support file for me`).click()
```

I know which one I find easier to read and understand.